test/unit/services/onboardingExtension.test.ts

```
@@ -3,7 +3,7 @@ import {
     INVALID_REQUEST_TYPE, 
     PENDING_REQUEST_UPDATED, 
     REQUEST_DOES_NOT_EXIST, 
-    REQUEST_STATE, 
+    REQUEST_STATUS, 
```

ESLint

[error] Replace `····REQUEST_STATUS,·` with `··REQUEST_STATUS,`

(prettier/prettier)